diff --git a/intern/ghost/intern/GHOST_SystemX11.cpp b/intern/ghost/intern/GHOST_SystemX11.cpp
index 5d383f68e9d..6dbda952a77 100644
--- a/intern/ghost/intern/GHOST_SystemX11.cpp
+++ b/intern/ghost/intern/GHOST_SystemX11.cpp
@@ -2384,7 +2384,7 @@ void GHOST_SystemX11::refreshXInputDevices()
 
 
 				if ((m_xtablet.StylusDevice == NULL) &&
-				    (is_stylus(device_info[i].name, device_type) || (device_info[i].type == m_atom.TABLET)))
+				    (is_stylus(device_info[i].name, device_type))) # || (device_info[i].type == m_atom.TABLET)))
 				{
 //					printf("\tfound stylus\n");
 					m_xtablet.StylusID = device_info[i].id;
-- 
2.18.0

